sandbox-web/src/main/java/com/exadel/sandbox/team5/ReviewRestController.java

```diff
@@ -24,5 +27,11 @@
             throw new ResponseStatusException(HttpStatus.NOT_FOUND, "Unknown discount id");
         }
     }
+
+    @GetMapping("/{id}/discounts")
+    public Page<Review> getByDiscountID(@PathVariable Long id) {
+        Pageable pageWithFiveElements = PageRequest.of(0, 5);
```

Comment:
Let's use Pageable object as part of SearchCriteria object in POST search REST methods